datafusion/core/src/datasource/physical_plan/file_stream.rs

```diff
@@ -269,7 +267,7 @@ impl<F: FileOpener> FileStream<F> {
             file_iter: files.into(),
             projected_schema,
             remain: config.limit,
-            file_reader,
+            file_opener: file_reader,
```

Comment:
Perhaps you can change the name of the parameter to this function to `file_opener` to match the field name too?

Reply:
Sorry I missed this one:D